module/modmanager/manager.go

```diff
@@ -446,35 +454,6 @@ func (mgr *Manager) newOnUnexpectedExitHandler(mod *module) func(exitCode int) b
 			return false
 		}
 
-		// If mod.handles was never set, the module was never actually ready in the
```

Comment:
I think we no longer need this section. Now, if `checkReady` was not successful (`mod.handles` was never set), we will have already called `Stop` on the process, and the `onUnexpectedExitHandler` will no longer be called for an "immediate crash".